upstairs/src/upstairs.rs

```diff
@@ -352,6 +358,11 @@ impl Upstairs {
             Some(d) => RegionDefinitionStatus::ExpectingFromDownstairs(d),
         };
 
+        let pool = rayon::ThreadPoolBuilder::new()
+            .num_threads(16)
```

Comment:
Pull this out into a named constant with some justification for its value?

Reply:
Done, although the number is admittedly a made-up value until I do more benchmarking.

Reply:
Yep!  Gotta start somewhere.